CMakeLists.txt

```diff
@@ -87,6 +87,20 @@ find_package(MPI REQUIRED)
 
 find_package(FFTW3 REQUIRED COMPONENTS DOUBLE MPI)
 
+if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+    set(FANS_LIB_DEFAULT OFF)
+endif ()
+option(FANS_LIB "Building FANS as a library to be used by the Micro Manager." ${FANS_LIB_DEFAULT})
```

Comment:
The name of the option `FANS_LIB` might be confusing, since FANS itself is built as a shared library (`libFANS.so`). I suggest to rename the option to something more specific, like `FANS_PYTHON_LIBRARY` or `FANS_MICRO_MANAGER` or similar.